src/rollup-plugin-deno-resolver/loadUrl.ts

```diff
@@ -22,9 +23,16 @@ export async function loadUrl(
     }
     case "http:":
     case "https:": {
-      const res = await fetch(url.href, fetchOpts);
+      try {
+        // get from deno cache storage
```

Comment:
```suggestion
```

Think the code speaks for itself :smile:

Reply:
> Think the code speaks for itself 😄

i think so 😂